### DISCUSSION TITLE
Magento DOB - datepicker / calender not working

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#78] [1. Help]
Hi,

As I am using breeze theme in my website I am getting below error when enabled DOB field on signup page from backend customer attributes. 
![image](https://user-images.githubusercontent.com/9527439/227482665-52ab08cd-c0d3-4a47-9390-b14c938af722.png)

![image](https://user-images.githubusercontent.com/9527439/227482932-e7812fd0-60de-4e18-a159-ff720e5df233.png)

how can I fix this. is there any other way to implement/override it in breeze js ?

Thanks

**Comment by vovayatsyuk** [comment]
We didn't implement the calendar in Breeze yet. For now, you have to do it on your own. I recommend using https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

**Reply by rutulritz** [reply]
Ok @vovayatsyuk let me check once

**Reply by rutulritz** [reply]
Hi @vovayatsyuk ,

I checked it and implemented. but I want to change date format as 'M-d-yy' (or in any format). 

How can I do this, I refered its document but not found the solution as it is browser specific.

Please help.

Thanks

**Reply by vovayatsyuk** [reply]
Unfortunately, the native date field doesn't allow to change date format. I will start working on date field implementation for Breeze and maybe it will be added in one of the next releases.

**Comment by vovayatsyuk** [comment]
Hi! I've just tagged Breeze 2.5.0 with calendar support. Please update and try it out.
